cypress/integration/page.spec.js

```diff
@@ -1183,18 +1274,26 @@
       page.waitForRequest('@commentsPostRequest');
 
       newCommentForm.nameInput().type('{selectAll}{backspace}Misha Hrynko');
-      newCommentForm.emailInput().type('{selectAll}{backspace}misha@mate.academy');
+      newCommentForm
+        .emailInput()
+        .type('{selectAll}{backspace}misha@mate.academy');
```

Comment:
Ensure that the `cy.get` and `its` statements are properly chained and terminated with semicolons. This ensures proper execution of the assertions.